src/util/tubemap.js

```diff
@@ -874,20 +878,35 @@ function setOccupiedUntil(map, read, pathIndex, y, node) {
 
 // compare read segments which are outside of nodes
 // by the y-coord of where they are coming from
```

Comment:
This comment isn't true anymore and needs to be updated.

```suggestion
// Compare read segments which are outside of nodes, to sort them in a good horizontal
// and then vertical display order.
```